docs/user_guide.md

```diff
@@ -5,16 +5,20 @@
 4. Profit!
 
 ## Dependencies
-1. Git
-2. Gradle
+1. **JDK `1.8.0_60`**  or later <br>
+> This app only works with Java 8 version. <br>
+2. **Git** on the command line <br>
+> Ensure that you're able to use it on the OS terminal. <br>
+
 ## How to Generate Dashboard
-```sh
-$ gradle clean build
-```
-Unzip the file in build/distributions/
+1. Download the latest executable Jar on our [release](https://github.com/reposense/RepoSense/releases/latest).
+2. Execute it on the OS terminal. <br>
+Usage: `java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY]`
+3. The dashboard can be in the folder designated in OUTPUT_DIRECTORY, or current working directory otherwise, as index.html.
+
+Sample usage:
 ```
-$ cd build/distributions/RepoSense/bin
-$ ./RepoSense -config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017
+$ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017
```

Comment:
shall we document both 'run from executable' and 'build from source' documentation?

Reply:
hmm ok